packages/astro/src/content/utils.ts

```diff
@@ -358,3 +387,90 @@ function search(fs: typeof fsMod, srcDir: URL) {
 	}
 	return { exists: false, url: paths[0] };
 }
+
+export async function getStringifiedLookupMap({
```

Comment:
Would it be possible to document this function a bit?

Reply:
Sure! Will also move to the `vite-plugin-content-virtual-mod` file where it's used; confusing to make a reusable util